Frends.Oracle.ExecuteQuery/Frends.Oracle.ExecuteQuery/Definitions/Result.cs

```diff
@@ -0,0 +1,17 @@
+﻿namespace Frends.Oracle.ExecuteQuery.Definitions
+{
+    /// <summary>
+    /// Return object with private setters.
+    /// </summary>
+    public class Result
+    { 
+        /// <summary>
+        /// Boolean value of success of the query 
+        /// </summary>
+        /// <example>true</example>
+        public bool Success { get; set; }
+
+        internal Result()
```

Comment:
Not sure about this, but I don't think you need constructor for this class, since you don't create object of this class. Ignore this if it is necessary.

Reply:
Removed empty constructor